src/providers/types/ethereum.d.ts

```diff
@@ -3,26 +3,36 @@ import { BaseConfig } from '../../constants';
 import BloctoProviderInterface from './blocto.d';
 
 export declare interface EthereumProviderConfig extends BaseConfig {
-  chainId: string | number | null;
-  rpc?: string;
-  server?: string;
+  chainId: string | number | null
+  rpc?: string
+  server?: string
 }
 
 export interface EIP1193RequestPayload {
-  id?: number;
-  jsonrpc?: string;
-  method: string;
-  params?: Array<any>;
+  id?: number
+  jsonrpc?: string
+  method: string
+  params?: Array<any>
 }
 
 export declare interface EthereumProviderInterface extends BloctoProviderInterface, IEthereumProvider {
-  chainId: string | number;
-  networkId: string | number;
-  chain: string;
-  net: string;
-  rpc: string;
-  server: string;
-  accounts: Array<string>;
-  request(args: EIP1193RequestPayload): Promise<any>;
+  chainId: string | number
+  networkId: string | number
+  chain: string
+  net: string
+  rpc: string
+  server: string
+  accounts: Array<string>
+  request(args: EIP1193RequestPayload): Promise<any>
+  loadSwitchableNetwork(
+    networkList: {
+      chainId: string
+      rpcUrls?: string[]
+    }[]
+  ): Promise<null>
```

Comment:
do we need add `checkAndAddNetwork` method.

Reply:
I think `loadSwitchableNetwork` is enough for use